src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -256,6 +256,7 @@ export const calculateBucketPrices: CalculateBucketPrices = args => {
 
         if (!assetIds.includes(asset)) return
         if (!includeTx) return
+        if (tx.status === chainAdapters.TxStatus.Failed) return
```

Comment:
ooo, very nice catch. looks like we consider the fees above, before ignoring this amount, so this looks good to me.